Makefile

```
@@ -1,4 +1,5 @@
 flash:
+	go run cmd/main.go -conf=tinygo
```

I suggest adding a default value in the flags on Go code with tinygo conf.

```go
const (
	gopherconEU22Logo = "./cmd/assets/gopherconeu-2022.jpg"
	gopherconUK22Logo = "./cmd/assets/gopherconuk-2022.jpg"
	gopherconUS22Logo = "./cmd/assets/gopherconus-2022.jpg"
	tinygoLogo        = "./cmd/assets/tinygo.jpg" // add the const
)

func main() {
	conf := flag.String("conf", tinygoLogo, "Choose the conference logo you want to (e.g.: tinygo, gceu22, gcuk22, gcus22)") // add the tinygoLogo as default value
	flag.Parse()
```

And we can pass no conf here. What do you think?
```suggestion
	go run cmd/main.go
```

---

Sounds nice. :+1: